lib/robots/dor_repo/gis_delivery/load_raster.rb

```diff
@@ -18,25 +18,23 @@ def perform_work
           end
 
           normalizer.with_normalized do |tmpdir|
-            Dir.chdir(tmpdir) do
-              tiffn = Dir.glob('*.tif').first
-              raise "load-raster: #{bare_druid} cannot locate GeoTIFF: #{tmpdir}" if tiffn.nil?
-
-              # copy to geoserver storage
-              path = if Settings.geohydra.geotiff.host == 'localhost'
-                       Settings.geohydra.geotiff.dir
-                     else
-                       [Settings.geohydra.geotiff.host, Settings.geohydra.geotiff.dir].join(':')
-                     end
-              cmd = "rsync -v '#{tiffn}' #{path}/#{bare_druid}.tif"
-              logger.debug "Running: #{cmd}"
-              system(cmd, exception: true)
-
-              # copy statistics files (produced by CopyData#compute_statistics, as of Feb 2024)
-              cmd = "rsync -v '#{tiffn}'.aux.xml #{path}/#{bare_druid}.tif.aux.xml"
-              logger.debug "Running: #{cmd}"
-              system(cmd, exception: true)
-            end
+            tif_filename = Dir.glob("#{tmpdir}/*.tif").first
+            raise "load-raster: #{bare_druid} cannot locate GeoTIFF: #{tmpdir}" if tif_filename.nil?
+
+            # copy to geoserver storage
+            destination_path = if Settings.geohydra.geotiff.host == 'localhost'
+                                 Settings.geohydra.geotiff.dir
+                               else
+                                 [Settings.geohydra.geotiff.host, Settings.geohydra.geotiff.dir].join(':')
```

Comment:
this is the only uncovered code path in the change.  and fwiw, all 3 deployed envs (qa/stage/prod), as well as the laptop development env, use `localhost` for `Settings.geohydra.geotiff.host`.  so nothing actually hits this condition in practice (afaict).

additionally, all the above mentioned envs use a fully qualified path for `Settings.geohydra.geotiff.dir`.  so, the `Dir.chdir` change should've had no the `rsync` calls below that use this path.

Reply:
maybe that's an indication that we should get rid of this condition and the associated `Settings.geohydra.geotiff.host` config?  though at this point i'd slightly rather do that in a separate PR, since this one has dragged out for a bit 😄